cmd/bundle/validate.go

```diff
@@ -66,7 +65,23 @@ func newValidateCommand() *cobra.Command {
 
 			return nil
 		case flags.OutputJSON:
-			return renderJsonOutput(cmd, b, diags)
+			renderOpts := render.RenderOptions{RenderSummaryTable: false}
+			err1 := render.RenderDiagnostics(cmd.OutOrStderr(), b, diags, renderOpts)
```

Comment:
This must be written to stderr or we risk invalidating the JSON.

This is a good candidate for an acceptance test that enforces a split output.

Reply:
I believe `cmd.OutOrStderr()` is already stderr.  Unless the command has a custom writer defined.

Reply:
It is stderr and it is already checked - since we pipe stdout to jq it would complain and crash if non-json lines were there.

I think it's enough but if you feel strongly about this, I can add another check specifically for stderr.

Reply:
@denik you can use `cmd.ErrOrStderr()`. That should also work for the golden tests.

Reply:
Otherwise in our test runner the diagnostics would be printed to the stdout buffer, because of:
```
	cli.SetOut(stdoutW)
	cli.SetErr(stderrW)
```

Reply:
Thanks! Switched to ErrOrStderr().

Also added explicit stderr redirect in one case, to be sure.